File: src/VSServiceStack/ProjectTemplates/FSharpEmptyAspNet/ServiceModel/Hello.fs
namespace $safeprojectname$

open ServiceStack

[<CLIMutable>]
type HelloResponse = 
    { Result : string }

[<Route("/hello")>]
[<Route("/hello/{name}")>]
type Hello() = 
    interface IReturn<HelloResponse>
    member val Name : string = null with get, set

File: src/ServiceStackXS/ProjectTemplates/Files/Hello.cs
using ServiceStack;

namespace ${Namespace}
{
	[Route("/hello")]
	[Route("/hello/{Name}")]
	public class Hello : IReturn<HelloResponse>
	{
		public string Name { get; set; }
	}

	public class HelloResponse
	{
		public string Result { get; set; }
	}
}